lua/neorg/modules/core/esupports/metagen/module.lua

```diff
@@ -29,9 +29,18 @@ end
 local function get_timestamp()
     -- generate a ISO-8601 timestamp
     -- example: 2023-09-05T09:09:11-0500
-    local tz_offset = get_timezone_offset()
-    local h, m = math.modf(tz_offset / 3600)
-    return os.date("%Y-%m-%dT%H:%M:%S") .. string.format("%+.4d", h * 100 + m * 60)
+    --
+    timezone_config = module.config.public.timezone
```

Comment:
```suggestion
    local timezone_config = module.config.public.timezone
```

Be sure to make this a local variable here! :)